app/controllers/index.js

```diff
@@ -1,5 +1,16 @@
-function doClick(e) {
-    alert($.label.text);
+function providers(){
+	var providerListView = Alloy.createController('providerList').getView();
```

Comment:
TD: instead of storing the view in a variable; we should be able to do
`Alloy.createController('providerList').getView().open();`